common/protob/messages-benfen.proto

```
@@ -0,0 +1,87 @@
+message BenfenAddress {
+    optional string address = 1;                // Benfen address as hex-encoded string
+}
+
+/**
+ * Request: ask device to sign Benfen transaction
+ * @start
+ * @next BenfenSignedTx/BenfenTxRequest
+ */
+message BenfenSignTx {
+    repeated uint32 address_n = 1;         // BIP-32 path to derive the key from master node
+    required bytes raw_tx = 2;             // serialized raw transaction
```

Buf (1.47.2)

Field named %!q(MISSING) should not be required.

(FIELD_NOT_REQUIRED)